src/main/java/io/ncbpfluffybear/supserv/machines/AdvancedElectricComposter.java

```diff
@@ -43,6 +44,10 @@ protected void registerDefaultRecipes() {
                     new ItemStack(Material.DIRT)
             );
         }
+
+        registerRecipe(1, new ItemStack(Material.SOUL_SOIL, 1),
```

Comment:
You don't need to specify size if it is 1

Reply:
removed, i thought it would be better to stay consistent as multiplicity was specified before.